easy_biologic/lib/ec_lib.py

```diff
@@ -230,7 +230,33 @@ class ERange( Enum ):
     v5   = 1
     v10  = 2
     AUTO = 3
-
+    
+    
+class Bandwidth( Enum):
+    """
+    Bandwidths
+    """
+    BW1 = 1  # "Slow"
+    BW2 = 2
+    BW3 = 3
+    BW4 = 4
+    BW5 = 5  # "Medium"
+    BW6 = 6
+    BW7 = 7  # "Fast"
+    # NOTE: 8 and 9 only available for SP300 series

```

Comment:
It may be worth creating a separate enum for the SP300 with the additional option, then. What's your opinion?

Reply:
I think that makes sense. What would you call them? I'm not familiar with Biologic's product series or naming conventions. Maybe Bandwidth300Series for the SP-300 series, but what would you call the more basic series?